{"name": "test_setup_failed_3[20]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <test_setup.TestSetUp object at 0x10facbb80>.valid", "trace": "self = <test_setup.TestSetUp object at 0x10facbb80>, a_number = 20\n\n    @pytest.mark.parametrize(\"a_number\", [21, 20])\n    def test_setup_failed_3(self, a_number):\n        \"\"\"\n        test setup failed\n        :return: None\n        \"\"\"\n>       assert self.valid\nE       assert False\nE        +  where False = <test_setup.TestSetUp object at 0x10facbb80>.valid\n\ntests/tests_setup/test_setup.py:80: AssertionError"}, "description": "\n        test setup failed\n        :return: None\n        ", "parameters": [{"name": "a_number", "value": "20"}], "start": 1722077104760, "stop": 1722077104760, "uuid": "0eced207-592b-4c4d-b1a5-7d3893f31a8b", "historyId": "e3251792b91e55f3280798914b32938e", "testCaseId": "30c10739d2f58af286790dfae347cf47", "fullName": "tests.tests_setup.test_setup.TestSetUp#test_setup_failed_3", "labels": [{"name": "tag", "value": "setup"}, {"name": "tag", "value": "xdist_group(name='setup')"}, {"name": "parentSuite", "value": "tests.tests_setup"}, {"name": "suite", "value": "test_setup"}, {"name": "subSuite", "value": "TestSetUp"}, {"name": "host", "value": "Aymans-MacBook-Pro.local"}, {"name": "thread", "value": "12476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_setup.test_setup"}]}